src/main/java/duke/tasks/Todo.java

```diff
@@ -0,0 +1,34 @@
+package duke.tasks;
+
+/**
+ * Class to create a Todo.
+ */
+class Todo extends Task {
+
+    /**
+     * Constructor for the Todo class.
+     *
+     * @param name name of the todo.
+     */
+    public Todo(String name) {
+        super(name);
+    }
+
+    /**
+     * Return string representation of the todo task.
+     *
+     * @return String representation of the todo task.
+     */
+    @Override
+    public String toString() {
+        String checkBox;
+        char taskType1 = 'T';
```

Comment:
Similar to an above comment, would it be better to rename the variable to `taskTypeTodo` instead? 
```suggestion
        char taskTypeTodo = 'T';
```